src/views/Project/index.jsx

```diff
@@ -57,18 +73,16 @@ const tagReposMapping = repositories =>
     }) => ({
       variables: {
         search: {
-          keywords: ['good-first-bug'],
+          ...bugzillaSearchOptions,
           products: productsList(projects[project].products).length
```

Comment:
`productsList` seems to check if products have no components listed under them and return the product names in an array. Would something like `productsWithNoComponents` be a better name than `productsList`?

Also, have we considered the case of products with and without components?

_Example:_

```yaml
name: Bugzilla
description: Get involved with the [Bugzilla](https://wiki.mozilla.org/Bugzilla) project or [BMO](https://wiki.mozilla.org/BMO), Mozilla's custom version
products:
 - Testing
 - Bugzilla: ['Administration', 'Attachments & Requests', 'Bug Import/Export & Moving', 'Bugzilla-General', 'bugzilla.org', 'Creating/Changing Bugs', 'Documentation', 'Email Notifications', 'Extensions', 'QA Test Scripts', 'Query/Bug Lists', 'Testing Suite', 'User Accounts', 'User Interface', 'WebService', 'Whining']
 - bugzilla.mozilla.org: ['Administration', 'API', 'Bugzilla Tweaks', 'Developer Box', 'Extensions: AntiSpam', 'Extensions: BMO', 'Extensions: ComponentWatching', 'Extensions: MyDashboard', 'Extensions: Needinfo', 'Extensions: UserProfile', 'General', 'rbbz', 'Sandstone/Mozilla Skin', 'User Interface']
```

_Note: I added a `Testing` product without components._

Refreshing the page doesn't seem to always show me the full list of bugs. Sometimes I only get bugs for Testing and sometimes I get bugs for all Testing, Bugzilla, and bugzilla.mozilla.org.